converter/file_read.py

```diff
@@ -0,0 +1,245 @@
+import xml.etree.ElementTree as ET
+import pandas as pd
+import logging
+import argparse
+import os
+import sys
+import re
+
+from constants import *
+
+
+def converter(args):
+    """Main entry point of the application
+    """
+    # create a dictionary of all paths to the files
+    paths = find_xml_files(args.input_dir)
+    # read the addresses from the files
+    addresses = read_xml_files(args.region, paths)
+    # write to csv
+    write_to_csv(addresses, args.region, args.output_dir)
+
+
+def find_xml_files(input_dir):
+    """Find all the xml files in the given directory
+    """
+    keys = FILE_KEYS
+    paths = {}
+    for root, _, files in os.walk(input_dir):
+        for file in files:
+            if file.split('.')[-1] == 'xml':
+                m = re.search('^(.+?)[0-9]', file)
+                if m:
+                    key = m.group(1)
+                else:
+                    logging.warn(
+                        'The contents of file %s can not be read by this script', file)
+                if key in keys:
+                    keys.remove(key)
+                    paths[key] = os.path.join(root, file)
+                else:
+                    logging.warn(
+                        'The contents of file %s can not be read by this script', file)
+    if keys:
+        logging.error(
+            'File for data of type %s was not found in the input folder', keys)
+        sys.exit(1)
+    return paths
+
+
+def read_xml_files(region, paths):
+    """Read all XML files for the specified region
+    """
+    logging.info('Started reading XML files')
+
+    addresses = []
+
+    if region in ['all', 'brussels']:
```

Comment:
in this method region "all" is used a few times, but the argument parser in the main method seems to specify "belgium" ?